src/components/ui/GradientBorder.tsx

```diff
@@ -9,23 +9,35 @@ import classnames, {
   width,
 } from 'classnames/tailwind'
 
-const parentButtonWrapper = classnames(
-  borderRadius('rounded-full'),
-  backgroundImage('bg-gradient-to-r'),
-  padding('p-px'),
-  gradientColorStops('from-secondary', 'to-accent'),
-  boxShadow('shadow-2xl', 'hover:shadow-lg', 'active:shadow-button-active')
-)
-const innerButtonWrapper = classnames(
-  width('w-full'),
-  borderRadius('rounded-full'),
-  backgroundColor('bg-primary-dark')
-)
+const parentButtonWrapper = (smallRadius?: boolean, withoutShadow?: boolean) =>
+  classnames(
+    borderRadius(smallRadius ? 'rounded-2xl' : 'rounded-full'),
```

Comment:
How about doing this one time? Now you have same check in 2 functiona

Reply:
fixed, ty